chains/solana/contracts/programs/ccip-router/src/instructions/v1/offramp.rs

```diff
@@ -572,13 +567,16 @@ fn should_execute_messaging(logic_receiver: &Pubkey, remaining_accounts_empty: b
 ///
 /// # Arguments
 /// * `token_indexes` - start indexes of token pool accounts, used to determine ending index for arbitrary messaging accounts
-/// * `logic_receiver` - receiver address from x-chain message, used to validate `remaining_accounts`
 /// * `extra_args_accounts` - arbitrary messaging accounts from the x-chain message, used to validate `accounts`.
-/// * `remaining_accounts` - accounts passed via `ctx.remaining_accounts`. expected order is: [program, receiver, ...additional message accounts]
+/// * `remaining_accounts` - accounts passed via `ctx.remaining_accounts`. expected order is: [logic_receiver, token_receiver, ...additional message accounts]
+///
+/// # Return values
+//  * `logic_receiver` - In EVM receiver means the address that all the listed tokens will transfer to and the address of the message execution.
+//     In Solana the receiver is split into two:
+//     * Logic Receiver (returned here) is the Program ID of the user's program that will execute the message.
+//     * Token Receiver (not present here, part of Any2SVMRampMessage), is the address which the ATA will be calculated from.
```

Comment:
This method does not include anything related to token transfers
```suggestion
/// # Return values
//  * `logic_receiver` is the Program ID of the user's program that will execute the message.
//  * `msg_accounts` the list of accounts used for the arbitrary execution
```